packages/flutter/lib/src/widgets/debug.dart

```diff
@@ -255,7 +255,7 @@ bool debugCheckHasMediaQuery(BuildContext context) {
 ///  * alternative: provide additional advice specific to the situation,
 ///    especially an alternative to providing a Directionality ancestor.
 ///    For example, "Alternatively, consider specifying the 'textDirection'
-///    argument.". Should be a funny punctuated sentence.
+///    argument.". Should be a fully punctuated sentence.
```

Comment:
WE DON'T HAVE FUN HERE.

Reply:
I seriously considered leaving the original or having it say "funny and fully" or something but...